liquibase-standard/src/main/java/liquibase/command/core/CalculateChecksumCommandStep.java

```diff
@@ -20,22 +31,34 @@ public class CalculateChecksumCommandStep extends AbstractCommandStep {
     protected static final String[] COMMAND_NAME = {"calculateChecksum"};
 
     public static final CommandArgumentDefinition<String> CHANGELOG_FILE_ARG;
-    public static final CommandArgumentDefinition<String> CHANGESET_IDENTIFIER_ARG;
+    public static final CommandArgumentDefinition<String> CHANGESET_PATH_ARG;
 
-    public static final CommandResultDefinition<CheckSum> CHECKSUM_RESULT;
+    public static final CommandArgumentDefinition<String> CHANGESET_ID_ARG;
+
+    public static final CommandArgumentDefinition<String> CHANGESET_AUTHOR_ARG;
 
-    protected static final int CHANGESET_ID_NUM_PARTS = 3;
-    protected static final int CHANGESET_ID_AUTHOR_PART = 2;
-    protected static final int CHANGESET_ID_CHANGESET_PART = 1;
-    protected static final int CHANGESET_ID_CHANGELOG_PART = 0;
+    public static final CommandResultDefinition<CheckSum> CHECKSUM_RESULT;
 
 
     static {
         CommandBuilder builder = new CommandBuilder(COMMAND_NAME);
         CHANGELOG_FILE_ARG = builder.argument(CommonArgumentNames.CHANGELOG_FILE, String.class).required()
-                .description("The root changelog file").build();
-        CHANGESET_IDENTIFIER_ARG = builder.argument("changesetIdentifier", String.class).required()
-                .description("Changeset ID identifier of form filepath::id::author").build();
+                                    .description("The root changelog file").build();
+
+        CHANGESET_PATH_ARG = builder.argument("changeSetPath", String.class)
+                                    .required()
+                                    .description("Changelog path in which the changeset is included")
+                                    .build();
+
+        CHANGESET_ID_ARG = builder.argument("changesetId", String.class)
```

Comment:
Very minor thing, but let's rename this argument name to `changeSetId` to make it camel case and match with the two others.